components/mpas-albany-landice/driver/glc_comp_mct.F

```diff
@@ -892,6 +892,19 @@ subroutine glc_run_mct( EClock, cdata_g, x2g_g, g2x_g)!{{{
 !         call mpas_stream_mgr_reset_alarms(stream_manager, direction=MPAS_STREAM_INPUT, ierr=err_tmp)
```

Comment:
Also delete this block of comments starting with `In coupled mode, we don't want to use forcing from a file`, as they are no longer relevant.  You could add a warning that if any input ("forcing") files here contain fields that the coupler passes, the input file versions here will clobber the fields from the coupler.  And maybe a note about what fields we imagine needing to read in here.

Reply:
I've added the following:

```
         ! NOTE: If any input ("forcing") files here contain fields that the coupler also passes, 
         ! the input file versions here will clobber the fields passed from the coupler. 
```
 Offhand, I don't have a good feeling for what fields you are imagining ... sfc mass balance? Let me know if you have a list you think I should add.

Reply:
Thanks, that comment seems good